php/php.editor/src/org/netbeans/modules/php/editor/csl/SemanticAnalysis.java

```diff
@@ -952,12 +810,7 @@ private boolean isDeprecatedType(QualifiedName qualifiedName, int offset) {
             if (!isCancelled() && isResolveDeprecatedElements()) {
                 VariableScope variableScope = model.getVariableScope(offset);
                 QualifiedName fullyQualifiedName = VariousUtils.getFullyQualifiedName(qualifiedName, offset, variableScope);
-                for (TypeElement typeElement : getDeprecatedTypes()) {
-                    if (typeElement.getFullyQualifiedName().equals(fullyQualifiedName)) {
-                        isDeprecated = true;
-                        break;
-                    }
-                }
+                isDeprecated = getDeprecatedTypes().contains(fullyQualifiedName);
```

Comment:
Use `contains()` instead of a for-each loop. I think it's better.

Reply:
More readable, definitely.